themes/default/content/blog/new-improved-pulumi-service-user-experience/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "A New and Improved User Experience For the Pulumi Service"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-08-31
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test. 
+# Max length is 160 characters.
+meta_desc: Pulumi Service, the easiest way to use Pulumi’s open-source universal infrastructure as code, just got better with a new and improved user experience.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - susan-evans
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+
+It’s been nearly four years since the Pulumi Service console launched back in 2018. The [Pulumi Service]({{< relref "/product/pulumi-service" >}}) is the easiest way to use Pulumi’s open source, universal infrastructure as code SDK at scale and provides a fully managed experience. It handles infrastructure state and secrets, sets up SAML SSO, integrates with CI/CD pipelines, and enforces compliance rules.
+
+In these last four years, the Service’s features and capabilities have grown quickly. We’ve added features like stack tags, stack changes view, and resource visualizations. But throughout all these changes, the look and feel of the Service has largely stayed the same—until now!
+
+<!--more-->
+
+Today we’re excited to launch a set of improvements to the overall user experience of the Pulumi Service. These updates include:
+
+- Easier navigation around the Service
```

Comment:
This blog was originally written with a usability focus that has since changed to a product focus, so I have zero strong opinions about any of it. Feel free to suggest what you want here and I'll accept it.